group4/81966_Georgi_Gerginov/IT_Department/components/department/department.cpp

```diff
@@ -0,0 +1,83 @@
+#include "department.h"
+
+const Queue<Task*> Department::q_from_tasks() const {
+	Queue<Task*> from_tasks;
+
+	for(auto curr : m_tasks) {
+		from_tasks.push(curr);
+	}
+
+	return from_tasks;
+}
+
+const OrderedSet<Task*> Department::worked_on_tasks() const {
+	OrderedSet<Task*> worked_on;
+
+	for(auto curr : m_tasks) {
```

Comment:
I know it's a trend these days but I would personally avoid using auto - now looking at this I have no idea what is the type of variable i.e. data I'm working with.